src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const price = PRICE_PER_DAY * days;
```

Comment:
const TOTAL_PRICE = PRICE_PER_DAY * days;

1.Make names descriptive, so they explain the purpose of variables and functions. 
const = STABLE_CONST_NAME;